hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/compact/plan/generators/BaseHoodieCompactionPlanGenerator.java

```diff
@@ -82,18 +82,20 @@ public HoodieCompactionPlan generateCompactionPlan(String compactionInstant) thr
     // TODO - rollback any compactions in flight
     HoodieTableMetaClient metaClient = hoodieTable.getMetaClient();
     CompletionTimeQueryView completionTimeQueryView = new CompletionTimeQueryView(metaClient);
-    List<String> partitionPaths = FSUtils.getAllPartitionPaths(
+    List<String> allPartitionPaths = FSUtils.getAllPartitionPaths(
         engineContext, metaClient.getStorage(), writeConfig.getMetadataConfig(), metaClient.getBasePath());
 
     // filter the partition paths if needed to reduce list status
-    partitionPaths = filterPartitionPathsByStrategy(writeConfig, partitionPaths);
-    if (partitionPaths.isEmpty()) {
+    List<String> strategyFilterPartitionPaths = filterPartitionPathsByStrategy(writeConfig, allPartitionPaths);
+    LOG.info("Strategy: {} matched {} partition paths from all {} partitions",
```

Comment:
There is no need to change this class right?

Reply:
> There is no need to change this class right?

Yes! I have updated the code~